rocAL-setup.py

```diff
@@ -389,7 +389,9 @@ def ERROR_CHECK(waitval):
     os.system('(cd '+deps_dir+'; git clone https://github.com/Tencent/rapidjson.git; cd rapidjson; mkdir build; cd build; ' +	
             linuxCMake+' ../; make -j$(nproc); sudo make install)')
 
-    # libtar - https://github.com/tklauser/libtar.git 
```

Comment:
@LakshmiKumar23 @rrawther - can we use the official libtar rather than someones clone with a tag - https://repo.or.cz/libtar.git

```
git clone -b v1.2.20 https://repo.or.cz/libtar.git
```